src/preferences/dialog/dlgprefcontrolsdlg.ui

```diff
@@ -489,6 +489,23 @@
        </property>
       </widget>
      </item>
+     <item row="12" column="1" colspan="2">
+      <widget class="QCheckBox" name="checkBoxDoubleWidgetSize">
```

Comment:
For now this is a checkbox by design, because we "only" have the request for double the size. 
And I want to avoid to have to much degrees of freedom for now. 

If you thing it is useful to allow free scaling, we can consider to go for it. Today almost all is calculated on pixel base so that 1x and 2x works nice and fractional scaling is kind of blurry.
It now supports @2x suffix to avoid this issue.